pom.xml

```diff
@@ -13,7 +13,7 @@
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.222</jenkins.version>
+    <jenkins.version>2.222.4</jenkins.version>
```

Comment:
can we take the 2.222.1 or is there a limitation regarding version 12 of the bom that forces us to go to 2.222.4 ?

Reply:
No we can't use the 2.222.1 because the Credentials plugin needs at least the version 2.222.4 of Jenkins.